### ISSUE TITLE
[Feature]: several terminals in termux

### ISSUE THREAD

**Issue by andreygrozin** [termux/termux-app#3728]
### Feature description

It would be very nice to have the possibility to use several terminals at once, with some easy way to switch between them. Something like what's done by screen or tmux, but built in termux.
I tried to use screen and tmux. Both of them, in principle, work. But they emulate some other terminals. When I start emacs with color highlighting under screen (or tmux), I see something much less nice than when I start it directly in termux. The same happens when I ssh from termux to a remote linux host and run a program displaying colored text (e.g., alpine). When I do it from under screen or tmux, this color highlighting does not work properly.

### Additional information

[https://www.gnu.org/software/screen/](https://www.gnu.org/software/screen/)
[https://github.com/tmux/tmux/wiki](https://github.com/tmux/tmux/wiki)

**Comment by agnostic-apollo** [comment]
Swipe from left edge of screen towards the right.

**Comment by andreygrozin** [comment]
> Swipe from left edge of screen towards the right.
When I swipe from left edge of screen towards the right while running termux, nothing happens (android 12).

**Comment by agnostic-apollo** [comment]
Use `DRAWER` extra key if android's gesture navigation is causing problem.

https://wiki.termux.com/wiki/Touch_Keyboard#Extra_Keys_Row

**Comment by andreygrozin** [comment]
> Use `DRAWER` extra key if android's gesture navigation is causing problem.
> https://wiki.termux.com/wiki/Touch_Keyboard#Extra_Keys_Row

Thanks! I didn't know about configuring extra keys. Now I removed many ones I don't need (I use the hacker's keyboard, and already have ctrl, alt, tab, ...), added INS and DEL (I missed them very much - in the hacker's keyboard, I had to press FN, then press DEL, then press FN again - very inconvenient), and added DRAWER. It works. Now I can have several sessions - in one of them I usually have emacs, in one ssh to a linux server, and so on). Great! Many thanks.
